src/app/wallet/wallet.component.html

```diff
@@ -43,94 +43,160 @@ <h4 class="mb-3 text-uppercase">
+                                    <div class="transactions__single-element" 
+                                         [class.transactions__error] = "transaction.state !== 'MINED' || transaction.type === 'APPROVE_INVESTMENT'">
+                                        <div class="transactions__left">
+                                            <div class="left__desc">
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'INVEST'">
+                                                    Investment <span class="left__conjuctor">to</span>
+                                                </div>
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'DEPOSIT'">
+                                                    Deposit
+                                                </div>
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'WITHDRAW'">
+                                                    Withdraw
+                                                </div>
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'SHARE_PAYOUT'">
+                                                    Dividend <span class="left__conjuctor">from</span>
+                                                </div>
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'CANCEL_INVESTMENT'">
+                                                    Cancel investment <span class="left__conjuctor">to</span>
+                                                </div>
+                                                <div class="left__transaction-type" 
+                                                    *ngIf="transaction.type === 'APPROVE_INVESTMENT'">
+                                                    Approve Investment <span class="left__conjuctor">to</span>
+                                                </div>
+                                                {{ transaction.description }}
+                                            </div>
+                                            <div class="left__info">
+                                                <span class="info__state-mined" 
+                                                    *ngIf="transaction.state === 'MINED'">
+                                                    <span class="state__invested" 
+                                                            *ngIf="transaction.type === 'INVEST'">
+                                                        1% Invested -
```

Comment:
When something is hardcoded, like this 1%, a good practice is to set a comment starting with TODO, e.g.:

```
// TODO: change this when percentage will be available from API
```